integration-tests/src/test/java/org/apache/fineract/integrationtests/LoanApplicationApprovalTest.java

```diff
@@ -92,6 +92,31 @@ public void loanApplicationApprovedAmountLessThanProposedAmount() {
 
     }
 
+    /*
+     * Equal test case: Approved amount non zero is equal to proposed amount
+     */
```

Comment:
Can you please help me understand the nature of this test? 
- I dont see whether the new posting date was set for the loan product.
- I dont see any assertion anywhere either